### ISSUE TITLE
The pretrained weight

### ISSUE THREAD

**Issue by yifanjiang19** [yogeshgajjar/BDD100K-YOLOV3-tiny#1]
Hi, thanks for sharing your work. Could you please provide a pretrained weight on BDD-100k? Thanks!

**Comment by bahulkark** [comment]
Yes please @yogeshgajjar. Please, provide the trained weights on BDD-100k. Thanks !
